packages/two_dimensional_scrollables/test/tree_view/tree_test.dart

```diff
@@ -718,6 +718,101 @@ void main() {
       expect(treeView.treeNodeBuilder, isA<TreeViewNodeBuilder<String>>());
       expect(treeView.treeRowBuilder, isA<TreeViewRowBuilder<String>>());
     });
+
+    testWidgets(
```

Comment:
Similar to the framework change, this test does not fail when run without the code change. It should so it verifies the fix works as expected, and prevents us from regressing in the future.